frontend/app/players/page.js

```diff
@@ -0,0 +1,198 @@
+'use client';
+
+import { useState } from 'react';
+import useRequest from '@/utils/useRequest';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+
+export default function Players() {
+  const { response: players, error: playersError, loading: playersLoading, refetch } = useRequest('GET', '/api/players');
+  const { response: matches, error: matchesError, loading: matchesLoading } = useRequest('GET', '/api/matches');
+  const [playerName, setPlayerName] = useState('');
+  const [playerRequested, setPlayerRequested] = useState(null);
+  const playerStats = (!playersLoading && !matchesLoading) && calculatePlayerStats(players, matches);
+
+  async function AddPlayer(playerName) {
```

Comment:
I think adding a confirmation modal or something of the sort will be a good addition after the button is clicked in case of user error especially since there's no way to delete players... unless 👀?

Reply:
make issue